packages/mask/src/plugins/Savings/protocols/aurigami/pairs.ts

```diff
@@ -0,0 +1,54 @@
+import type Web3 from 'web3'
+import { ChainId, FungibleTokenDetailed, createNativeToken } from '@masknet/web3-shared-evm'
+import type { SavingsProtocol, ProtocolPairsResolver } from '../../types'
+import CompoundLikeFetcher from '../common/CompoundLikeFetcher'
+import AurigamiProtocol from './AurigamiProtocol'
+import type { PairConfig } from '../common/protocol/BenQiRewardProtocol'
+
+// https://app.aurigami.finance/assets/vendor.1efcd1de.js  search: 'COMPTROLLER:'
+export const AURIGAMI_COMPTROLLER = '0x817af6cfAF35BdC1A634d6cC94eE9e4c68369Aeb'
+export const AURIGAMI_ORACLE = '0xC6e5185438e1730959c1eF3551059A3feC744E90'
+export const AURIGAMI_LENS = '0xC41a5C1625d492436600789469c1CE2eA20CEA6B'
+
+const excludePairs = ['DAI']
+const TokenLogos: { [key: string]: string } = {
```

Comment:
We should generate the icon URL in the runtime. However, it will be a nightmare when more pairs get added.

Reply:
> We should generate the icon URL in the runtime. However, it will be a nightmare when more pairs get added.

for now, I'm gonna move them to `TokenIcon\TokenIconSpecialIconList.json`